usaspending_api/references/v2/views/recipients.py

```diff
@@ -87,16 +87,19 @@ def _create_es_search(self, search_text: str, recipient_levels: List[str], limit
         """
         es_recipient_search_fields = ["recipient_name", "uei", "duns"]
 
-        query = ES_Q(
-            "bool",
-            should=[
-                ES_Q("query_string", query=search_text, fields=es_recipient_search_fields),
-                ES_Q("match", recipient_name=search_text),
-                ES_Q("match", uei=search_text),
-                ES_Q("match", duns=search_text),
-            ],
-            minimum_should_match=1,
-        )
+        # query = ES_Q(
+        #     "bool",
+        #     should=[
+        #         ES_Q("query_string", query=f"*{search_text}*", fields=es_recipient_search_fields),
+        #         ES_Q("match", recipient_name=search_text),
+        #         ES_Q("match", uei=search_text),
+        #         ES_Q("match", duns=search_text),
+        #     ],
+        #     minimum_should_match=1,
+        # )
```

Comment:
Can you remove these lines since they've been commented out and replaced by lines 100-102?

Reply:
done. Pls review again